cmake/Dependencies.cmake

```diff
@@ -47,30 +84,23 @@ ELSE()
     SET(USE_CUDNN FALSE)
 ENDIF()
 
-IF(USE_CBLAS)
```

Comment:
Yes. USE_CBLAS would always be ON. If we remove it, then we need to remove other places using USE_CBLAS